docs/spark-queries.md

```diff
@@ -78,10 +78,10 @@ val df = spark.table("prod.db.table")
 Iceberg 0.11.0 adds multi-catalog support to `DataFrameReader` in both Spark 3.x and 2.4.
 
 Paths and table names can be loaded with Spark's `DataFrameReader` interface. How tables are loaded depends on how
-the identifier is specified. When using `spark.read.format("iceberg").path(table)` or `spark.table(table)` the `table`
+the identifier is specified. When using `spark.read.format("iceberg").load(table)` or `spark.table(table)` the `table`
```

Comment:
I could not find 'path' method on DataFrameReader, only 'load', so I assume its a typo, unless it was removed sometime ago.  Also later in the doc 'load' is used.